qutebrowser/browser/webengine/webview.py

```diff
@@ -238,3 +239,18 @@ def acceptNavigationRequest(self,
             is_main_frame=is_main_frame)
         self.navigation_request.emit(navigation)
         return navigation.accepted
+
+    def chooseFiles(
+        self,
+        mode: QWebEnginePage.FileSelectionMode,
+        old_files: Iterable[str],
+        accepted_mimetypes: Iterable[str],
+    ) -> List[str]:
+        """Override chooseFiles to (optionally) invoke custom file uploader."""
+        handler = config.val.fileselect.handler
+        if handler == "default":
+            return super().chooseFiles(mode, old_files, accepted_mimetypes)
+
+        assert handler == "external"
```

Comment:
```suggestion
        assert handler == "external", handler
```

So that the error printed is "AssertionError: someothervalue" rather than just "AssertionError" - i.e. Python prints the thing after the comma if the assertion fails, which helps with debugging issues:

```pycon
>>> handler = "notext"
>>> assert handler == "external"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
>>> assert handler == "external", handler
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: notext
```

(With pytest, we get nice informations for AssertionErrors without that, but that's a [pytest-specific feature](https://docs.pytest.org/en/stable/assert.html))

Reply:
Done